src/vm-repair/azext_vm_repair/_params.py

```diff
@@ -33,6 +33,7 @@ def load_arguments(self, _):
         c.argument('associate_public_ip', help='Option to create repair vm with public ip')
         c.argument('distro', help='Option to create repair vm from a specific linux distro (rhel7|rhel8|suse12|ubuntu20|centos7|oracle7)')
         c.argument('yes', help='Option to skip prompt for associating public ip and confirm yes to it in no Tty mode')
+        c.argument('no', help='Option to skip prompt for associating public ip and confirm no to it in no Tty mode')
```

Comment:
Instead of yes and no as separate options. Is it good to add one command as public ip address yes or no?

Reply:
I dont know if we have to maintain the backward compatibility of passing yes.

Reply:
I personally don't like the parameter name `--yes` or `--no`, and I agree with you that using a new parameter like `createPublicIp` makes more sense. However, for the sake of backward compatibility, I think it's better to use `--no` in this situation.

Reply:
+1. We cannot break backward compat. I also don't like --yes, --no. Another option could be to introduce a new param and have it enable yes and no. That way, users can use the new param to specify yes/no and the old value van remain with yes support only. We can update documentation to remove --yes over time.